cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/LanguageTraits.kt

```diff
@@ -206,4 +206,7 @@ interface HasShortCircuitOperators : LanguageTrait {
     val conjunctiveOperators: List<String>
     // '||', 'or', 'v'
     val disjunctiveOperators: List<String>
+
+    val operatorCodes: Set<String>
```

Comment:
Missing documentation - also for conjunctive and disjunctive, even if its not your code ;) I wonder if this is also misleading because these are only *boolean* operators, right? Not all the operator codes a language potentially has.

Reply:
Kinda, however if other operators are interpreted more loosely as a question "is this non null", then it is not an issue. I think these are the best terms we can use here but maybe we find another.